config/install/core.entity_form_display.node.landing_page_lb.default.yml

```diff
@@ -2,38 +2,95 @@ langcode: en
 status: true
 dependencies:
   config:
+    - entity_browser.browser.images_library
+    - field.field.node.landing_page_lb.field_meta_description
+    - field.field.node.landing_page_lb.field_meta_image
+    - field.field.node.landing_page_lb.field_meta_tags
     - field.field.node.landing_page_lb.layout_builder__layout
     - node.type.landing_page_lb
   module:
+    - field_group
+    - metatag
```

Comment:
added https://github.com/YCloudYUSA/y_lb/pull/122/commits/f2d89e4c65009965a309e218cc4a74bf525a36da
Only on composer level, because module could be disabled

Reply:
What is the case where metatag could be disabled? It's included with even the standard build of the distro so I'm not sure how it would be off.

Reply:
I see, my bad, I thought metatag is not enabled by default. 
All good here then